dexmaker-tests/src/androidTest/java/com/android/dx/stock/ProxyBuilderTest.java

```diff
@@ -984,7 +998,7 @@ private void doTestOrderClassWithDexMakerCaching() throws Exception {
         assertFalse(proxy.returnsBoolean());
         assertEquals(1.0, proxy.returnsDouble());
         assertNotNull(proxy.returnsObject());
-        assertEquals(2, versionedDxDir.listFiles().length);
+        assertTrue(versionedDxDir.listFiles().length != 0);
```

Comment:
Should we maybe add a second check here, to make sure there are jar files? Not sure if its worth it.